addons/payment/utils.py

```diff
@@ -6,6 +6,7 @@
 from odoo.http import request
 from odoo.tools import consteq, float_round, ustr
 from odoo.tools.misc import hmac as hmac_tool
+from odoo.addons.payment.const import CURRENCY_ISOCODE_EXPONENT
```

Comment:
As per the guidelines, module imports should be in a separate block.
```suggestion

from odoo.addons.payment.const import CURRENCY_ISOCODE_EXPONENT
```